pkg/api/universal/scheduler.go

```diff
@@ -24,9 +24,12 @@ import (
 )
 
 func (a *Universal) ScheduleJob(ctx context.Context, inReq *runtimev1pb.ScheduleJobRequest) (*emptypb.Empty, error) {
+	// TODO: make this specific for error metadata and dont send along to scheduler since these are now first class fields
+	// However, this change will need to be done once the 'triggerJob' from the cron library sends the data back
+	// So rm this once the cron lib sends the trigger job appID && namespace && scope back
 	metadata := map[string]string{
```

Comment:
We don't need this metadata anymore if appId and namespace are 1st class attributes.

Reply:
fair point, I can rm it here, but they still need to be appended to the metadata in scheduler then bc I need them for the lookup of the correct stream at trigger time, which rn I will only have metadata to use for the lookup.